css/responsive/desktop-mid.styl

```diff
@@ -1 +1,4 @@
-//copy styles from site.styl
\ No newline at end of file
+//copy styles from site.styl
+
+.container
+  width: 993px
```

Comment:
no. but what is desktop mid anyways?  

lets just leave this as-is and attend to it properly later.  the responsive approach needs some revision i think, no?